src/Workers/Ordinals/Parse.ts

```diff
@@ -98,7 +98,7 @@ async function commit(height: number) {
   log(`\n💽 Removed ${outpoints.toLocaleString()} outpoints [${ts.now} seconds]\n\n`);
 }
 
-function hasReachedTreshold(height: number) {
+function hasReachedThreshold(height: number) {
```

Comment:
Should this function be checking the threshold height? We haven't run this code for a while, so not entirely sure if it is working as it should.

Reply:
Ill add the check.

Reply:
I dont think this code is run at all since its importing a redis service which does not exist anywhere in the project.